app/services/make_user_admin.rb

```diff
@@ -0,0 +1,29 @@
+class MakeUserAdmin
+  def initialize(user_name)
+    @user_name = user_name
+  end
+
+  def call
+    user = User.with_username(@user_name).first
+    user.present? ? add_admin_role(user) : user_not_found_message
+  end
+
+  private
+
+  def user_not_found_message
+    'User not found in Supermarket.  Make sure the user exists in Supermarket before making it an admin.'
```

Comment:
Since we're pulling in the Rails environment in the Rake task, can we just use i18n tags for these instead?